### PR TITLE
Avatar badge components completed

### PR DESCRIPTION
**Pull request by ravikumar0403**

### CODE REVIEW

**Review by sonishreyas — COMMENTED** [review]
@ravikumar0403 Looks good only change I think can be done is that all units can be converted to rems. But it's not necessary.

File: common/colors.css [hunk 1]
```diff
@@ -10,6 +10,10 @@
     --bg-color: #fcfcfc;
     --bg-secondary : #ffffff;
 
+    --red : #f87171;
+    --green : #34d399;
+    --yellow : #facc15;
+
     /* --primary-color : #03A9F4;
     --primary-dark : #B3E5FC;
     --primary-light : #0288D1;
```
Comment (vidhi-thakur) [inline common/colors.css:20]: I suggest naming your variables as `--primary-color` or `--secondary-color` like you did in your commented code. Also, you can delete the commented code if it is not required.

**Comment by ravikumar0403** [comment]
> @ravikumar0403 Looks good only change I think can be done is that all units can be converted to rems. But it's not necessary.

I missed that part. Thanks @sonishreyas I'll make the change now

File: common/colors.css (hunk 1 above)
Comment (ravikumar0403) [reply common/colors.css:20]: Thanks @vidhi-thakur I'll update them with better names.
and I'll be using the commented colors for dark mode later.

**Pull request merged**